src/flownet/realization/_schedule.py

```diff
@@ -325,7 +325,7 @@ def get_keywords(
         particular keyword class.
 
         Args:
-            date: Date at which to lookup keywords
+            date: Date or list of dates at which to lookup keywords
```

Comment:
Update docstring to "dates". Also in the descriptiona few lines above it should be changed.

"at a given date" isn't correct anymore.